config/routes.rb

```diff
@@ -29,6 +30,14 @@
       post :select
     end
 
+    #JE Todo: Confirm if we should nest these internal API routes under a namespace like "API::V1".
+    #JE Todo: Confirm if there is a better way to define these routes.
+    namespace :calendar_events do
+      root to: :index
+      get 'loan/:loan_id', to: :loan
+      get 'division/:division_id', to: :division
+    end
```

Comment:
I think `/calendar_events?loan_id=123` or just `/calendar_events` (for the general calendar case) is fine for now. Don't want to overcomplicate things. We are not passing division specific stuff to any of the other index controllers so I don't want to do it here.